packages/apollo-server-cache-memcached/src/__tests__/tsconfig.json

```diff
@@ -0,0 +1,8 @@
+{
+  "extends": "../../../../tsconfig.test.base",
+  "include": ["**/*"],
+  "references": [
+    { "path": "../../tsconfig.json" },
```

Comment:
Just out of curiosity, the explicit `tsconfig.json` files aren't necessary, right? If they were left off it would work the same?  (Though it seems to make sense to be explicit since we occasionally use `tsconfig.test.json`?)

Reply:
Yes, that's true. I originally used `tsconfig.build.json` at the package-level as well, so we could have a `tsconfig.json` with references to both the build and test project for a package. That didn't work however, because the TypeScript language server (and thus VS Code) doesn't currently follow `references` when looking up a project config for a file.